package.json

```diff
@@ -161,10 +161,23 @@
         },
         "category": "PowerShell"
       },
+      {
+        "command": "PowerShell.Run",
```

Comment:
@jpogran is there a reason this can't just be `workbench.action.debug.start`?

Reply:
No, I just had it in my head you need to make a command for anything you were going to use 😅 . Works fine as you suggest